src/frontend/components/Chat/StudentChat/index.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import { act, render, screen } from '@testing-library/react';
+import React from 'react';
+import { wrapInIntlProvider } from '../../../utils/tests/intl';
+import { useMessagesState } from 'data/stores/useMessagesStore';
+import { DateTime } from 'luxon';
+import { StudentChat } from '.';
+import { converse } from '../../../utils/window';
+import userEvent from '@testing-library/user-event';
+
+const sendMessageMock = (msgContent: string) =>
+  act(() =>
+    useMessagesState.getState().addMessage({
+      dateTime: DateTime.fromISO('2020-01-01T12:12:12'),
+      sender: 'John Doe',
+      content: msgContent,
+    }),
+  );
+
+jest.mock('../../../utils/window', () => ({
+  converse: {
+    sendMessage: jest.fn(sendMessageMock),
+  },
+}));
+
+describe('<StudentChat />', () => {
+  it('checks that a message is displayed after having been send with the input bar', async () => {
+    jest.useFakeTimers();
```

Comment:
```suggestion
    jest.useFakeTimers();
```
ditto

Reply:
I'm not sure to understand what you want to point out here. Maybe multiple rebase has changed your selectioned lines ?